daemon/src/lib.rs

```diff
@@ -310,6 +319,7 @@ where
             _archive_failed_cfds_actor: archive_failed_cfds_actor,
             _tasks: tasks,
             maker_online_status_feed_receiver,
+            _pong_actor: pong_address,
```

Comment:
❓How did this ever work, if we dropped `pong_address`? - probably it was fine because endpoint never stopped 😅

Reply:
yeah, the endpoint never stopped, and it contains a `StrongMessageChannel`, that increases the refcount.